scripts/ci/determine-tests.mjs

```diff
@@ -176,13 +176,16 @@ const changedFiles = getChangedFiles(base, head).split(EOL);
 const outputNameTestsToRun = process.argv[2];
 const outputNameShardIndex = process.argv[3];
 const outputNameShardTotal = process.argv[4];
+const runAllTests = process.argv[5] === 'false';
 const projectRoot = process.env.projectRoot;
 const atomicSourceComponents = join('packages', 'atomic', 'src', 'components');
 
 try {
   const testFiles = findAllTestFiles(atomicSourceComponents);
   const testDependencies = createTestFileMappings(testFiles, projectRoot);
-  const testsToRun = determineTestFilesToRun(changedFiles, testDependencies);
+  const testsToRun = runAllTests
+    ? testFiles.join(' ')
+    : determineTestFilesToRun(changedFiles, testDependencies);
```

Comment:
This will only work for the basic use case where tests need to be determined. The NoRelevantChangesError error below won't run all tests all the time in the merge queue.